certified-connectors/FactSet/apiDefinition.swagger.json

```diff
@@ -1,1411 +1,1422 @@
-                  "description": "Book ID"
-                }
-              }
-            }
-          }
-        },
-        "summary": "BookBuilder - Create Book from Template",
-        "description": "This endpoint retrieves book status, book name, and book ID for ticker requested in JSON format. This end-point accepts ticker and template_id as inputs. If the template_id input is not used, a book will be created with FactSet's default template.",
-        "operationId": "CreateBookFromTemplate",
-        "parameters": [
-          {
-            "name": "body",
-            "in": "body",
-            "required": false,
-            "schema": {
-              "type": "object",
-              "properties": {
-                "ticker": {
-                  "type": "string",
-                  "description": "Ticker"
-                },
-                "template_id": {
-                  "type": "string",
-                  "description": "Template ID"
-                }
-              }
-            }
-          }
-        ]
-      }
-    },
-    "/book-builder-api/v1/download-api-book/{book_id}": {
-      "get": {
-        "responses": {
-          "200": {
-            "description": "Success",
-            "schema": {
-              "type": "object",
-              "properties": {}
-            }
-          }
-        },
-        "summary": "BookBuilder - Get PDF",
-        "description": "This endpoint will return the PDF output given a book_id.",
-        "operationId": "GetPDF",
-        "parameters": [
-          {
-            "name": "book_id",
-            "in": "path",
-            "required": true,
-            "type": "string",
-            "x-ms-summary": "Book ID",
-            "description": "ID of the book to download"
-          }
-        ]
-      }
-    }
-  },
-  "definitions": {},
-  "parameters": {},
-  "responses": {},
-  "securityDefinitions": {
-    "oauth2_auth": {
-      "type": "oauth2",
-      "flow": "accessCode",
-      "authorizationUrl": "https://auth.factset.com/as/authorization.oauth2",
-      "tokenUrl": "https://auth.factset.com/as/token.oauth2",
-      "scopes": {}
-    }
-  },
-  "security": [
-    {
-      "oauth2_auth": []
-    }
-  ],
-  "tags": []
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "FactSet",
+    "description": "Automate your workflows using the FactSet Connector to stay on top of relevant news and events for companies of interest, highlight important topics to drive your discussions, prepare for meetings, and more.",
+    "version": "1.0"
+  },
+  "host": "api.factset.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/book-builder-api/book-list": {},
+    "/book-builder-api/template-list": {},
+    "/book-builder-api/get-book-id/{ticker}": {},
+    "/signals/v2/events/headlines": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
```

Comment:
@KyleVErwin The 'default' response should not have schema definition. Schemas should be defined on expected responses only. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses). Please fix all responses.
```
##[error]The 'default' response should not have schema definition. Schemas should be defined on expected responses only. ValuePath: /paths/~1signals~1v2~1events~1headlines/get/responses/default
##[error]The 'default' response should not have schema definition. Schemas should be defined on expected responses only. ValuePath: /paths/~1signals~1v2~1events~1details/get/responses/default
##[error]The 'default' response should not have schema definition. Schemas should be defined on expected responses only. ValuePath: /paths/~1signals~1v2~1events~1adaptive-cards/get/responses/default
##[error]The 'default' response should not have schema definition. Schemas should be defined on expected responses only. ValuePath: /paths/~1book-builder-api~1v1~1book-list/get/responses/default
##[error]The 'default' response should not have schema definition. Schemas should be defined on expected responses only. ValuePath: /paths/~1book-builder-api~1v1~1template-list/get/responses/default
##[error]The 'default' response should not have schema definition. Schemas should be defined on expected responses only. ValuePath: /paths/~1book-builder-api~1v1~1create-book-from-template/post/responses/default
##[error]The 'default' response should not have schema definition. Schemas should be defined on expected responses only. ValuePath: /paths/~1book-builder-api~1v1~1download-api-book~1{book_id}/get/responses/default
##[error]The 'default' response should not have schema definition. Schemas should be defined on expected responses only. ValuePath: /paths/~1cognitive~1nlp~1v1~1ner~1entities/post/responses/default

```